File: Algorithms/01-Warmup/03-a-very-big-sum/03.rb
#!/usr/bin/env ruby

def aVeryBigSum(ar)
    total = 0
    ar.each do |number|
        total += number
    end
    total
end

n = gets.strip.to_i
ar = gets.strip
ar = ar.split(" ").map(&:to_i)
result = aVeryBigSum(ar)
puts result

File: Algorithms/01-Warmup/03-a-very-big-sum/03.py
#!/usr/bin/env python


def aVeryBigSum(ar):
    total = 0
    for number in ar:
        total += number
    return total


n = int(input().strip())
ar = list(map(int, input().strip().split(' ')))
result = aVeryBigSum(ar)
print(result)
